### PR TITLE
[3/3] Add remaining tasks & tutorials, add driver docs

### PR DESCRIPTION
**Pull request by tstromberg**
Please pay special attention to the rewritten driver docs.

Preview URL: https://deploy-preview-4989--kubernetes-sigs-minikube.netlify.com/docs/reference/drivers/

### CODE REVIEW

**Comment by afbjorklund** [comment]
`7:01:01 AM: Error: Error building site: "/opt/build/repo/site/content/en/_index.html:1:1": timed out initializing value. This is most likely a circular loop in a shortcode`

File: site/content/en/docs/Tasks/loadbalancer.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Tunnel might ask you for password for creating and deleting network routes.
+
+### DNS resolution
+
+If you are on macOS, the tunnel command also allows DNS resolution for Kubernetes services from the host.
+
+### Cleaning up orphaned routes
+
+If the `minikube tunnel` shuts down in an unclean way, it might leave a network route around.
```
Comment (sanguinecloud) [inline site/content/en/docs/Tasks/loadbalancer.md]: 'shuts down in an unclean way' > 'shuts down abruptly' (or unexpectedly)?

File: site/content/en/docs/Tasks/loadbalancer.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+This case the ~/.minikube/tunnels.json file will contain an entry for that tunnel.
+To cleanup orphaned routes, run:
```
Comment (sanguinecloud) [inline site/content/en/docs/Tasks/loadbalancer.md]: clean up

File: site/content/en/docs/Tasks/loadbalancer.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+````shell
+minikube tunnel --cleanup
+````
+
+### Avoid entering password multiple times
```
Comment (sanguinecloud) [inline site/content/en/docs/Tasks/loadbalancer.md]: passwords

File: site/content/en/docs/Tasks/loadbalancer.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+`minikube tunnel` runs as a separate daemon, creates a network route on the host to the service CIDR of the cluster using the cluster's IP address as a gateway. Adding a route requires root privileges for the user, and thus there are differences in how to run `minikube tunnel` depending on the OS.
```
Comment (sanguinecloud) [inline site/content/en/docs/Tasks/loadbalancer.md]: creates > creating

File: site/content/en/docs/Tutorials/continuous_integration.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+export CHANGE_MINIKUBE_NONE_USER=true
+export KUBECONFIG=$HOME/.kube/config
+
+mkdir -p $HOME/.kube $HOME/.minikube
+touch $KUBECONFIG
+
+sudo -E minikube start --vm-driver=none
+```
+
+## Troubleshooting
```
Comment (sanguinecloud) [inline site/content/en/docs/Tutorials/continuous_integration.md]: A rogue heading!

File: site/content/en/docs/Reference/Drivers/_index.md [hunk 6]
```diff
@@ -3,6 +3,8 @@ title: "Drivers"
 linkTitle: "Drivers"
 weight: 8
 date: 2017-01-05
+date: 2018-08-05
 description: >
-  Configuring minikube drivers
+  Configuring various minikube drivers
```
[the remark this answers is not in the document]
Comment (tstromberg) [reply site/content/en/docs/Reference/Drivers/_index.md]: Intentional branding. =)

File: site/content/en/docs/Tasks/dashboard.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+The Dashboard is a web-based Kubernetes user interface. You can use it to:
+
+
+- deploy containerized applications to a Kubernetes cluster
+- troubleshoot your containerized application
+- manage the cluster resources
+- get an overview of applications running on your cluster
+- creating or modifying individual Kubernetes resources (such as Deployments, Jobs, DaemonSets, etc)
+ 
+For example, you can scale a Deployment, initiate a rolling update, restart a pod or deploy new applications using a deploy wizard.
```
[the remark this answers is not in the document]
Comment (tstromberg) [reply site/content/en/docs/Tasks/dashboard.md]: Deployment is a proper Kubernetes API object name: https://cloud.google.com/kubernetes-engine/docs/concepts/deployment

**Pull request merged**